Client/Ecosia/UI/MyImpact/MyImpactCell.Indicator.swift

```diff
@@ -9,8 +9,8 @@ extension MyImpactCell {
         var value = Double() {
             didSet {
                 let guide = CGMutablePath()
-                guide.addArc(center: .init(x: 120, y: 120),
-                             radius: 112,
+                guide.addArc(center: .init(x: bounds.size.width/2, y: bounds.size.width/2),
```

Comment:
I like it (though I was under the impression NTP's counter didn't have `Indicator`, only `Progress`.)

Reply:
That is true, it does not have the Indicator. I only noticed that later. Still I thought it's helpful that the Indicator is also configurable from the outside, don't you think?

Reply:
We might never need to have a different configuration. But it is already done, so no harm on leaving it 👍